tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts

```diff
@@ -38,6 +38,12 @@ export default {
                 activate_identity: "(LitentryIdentity, LitentryIdentity, LitentryIdentity, H256)",
                 request_vc: "(LitentryIdentity, LitentryIdentity, Assertion, Option<RequestAesKey>, H256)",
                 set_identity_networks: "(LitentryIdentity, LitentryIdentity, LitentryIdentity, Vec<Web3Network>, H256)",
+                __Unused_link_identity_callback: "Null",
+                __Unused_request_vc_callback: "Null",
+                __Unused_handle_imp_error: "Null",
+                __Unused_handle_vcmp_error: "Null",
+                __Unused_send_erroneous_parentchain_call: "Null",
+                __Unused_remove_identity: "Null",
```

Comment:
If you don't need them in ts-test, you don't need to add it

Reply:
yep now I'm a bit lost on using __Unused definitions

Reply:
Use it if you want to keep indices unchanged while removing invariant or if invariant is feature toggled.
Otherwise generated ts types wouldn't match indices on server side causing integration problems.

Reply:
You could refer to https://docs.substrate.io/reference/scale-codec/ for more info

Reply:
yes, that's what I understood from your initial proposition, but what if `link_identity_callback` starts on index = 20? shall we insert 14 empty records to sync the index?

Reply:
for ts yes :)
those callbacks are supposed to be called internally (e.g. from another trusted call), so we don't expect them to be called by ts directly.

Reply:
If we don't want to reflect some calls in generated types because we don't expect consumers to dispatch them then it's a good reason to use `__Unused` too as Kai described.

Reply:
hm, from what I understood we also don't plan to call `remove_identity` by ts, isn't it? Also what about original integritee trusted calls, starting from index 50, are they also called internally?

Reply:
both answers would be no.

However, `remove_identity` is more like triggerable from externals (rust-cli or ts), so conventionally it should follow `set_identity_networks` - that's why I suggested `index 5`

Reply:
> original integritee trusted calls, starting from index 50, are they also called internally - no

Does it mean we should fill empty indices with `__Unused: 'Null'`? @kziemianek

Reply:
Current code is ok, I explained it a bit in this [comment](https://github.com/litentry/litentry-parachain/pull/2268/files#r1415166487)